src/app/vault/add-edit.component.html

```diff
@@ -85,7 +88,7 @@ <h2 class="modal-title" id="cipherAddEditTitle">{{title}}</h2>
                             </div>
                         </div>
                     </div>
-                    <div class="row">
+                    <div class="row" *ngIf="!cipher.isDeleted">
```

Comment:
Why do we disable some things, but here you hide it?

Reply:
Input fields, select lists, etc. would be disabled (read-only) where actions (add/remove) or rows with the entire purpose of adding new items like Urls or Custom fields, etc. are completely hidden/removed from view. We can discuss the general UX approach tomorrow so any adjustments can be made before we move on to other clients.

Reply:
Fixed